app/src/main/java/com/breadwallet/presenter/history/HistoryFragment.kt

```diff
@@ -43,8 +43,8 @@ class HistoryFragment :
         return binding.root
     }
 
-    override fun onActivityCreated(savedInstanceState: Bundle?) {
-        super.onActivityCreated(savedInstanceState)
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```

Comment:
change to `onViewCreated` since `onActivityCreated` deprecated

Reply:
nice!